src/word_search_generator/config.py

```diff
@@ -1,7 +1,7 @@
 from enum import Enum, unique
 
 # puzzle settings
-min_puzzle_size = 10
+min_puzzle_size = 5
```

Comment:
There's the most-actionable item from #21 handled. Think that Issue ought to be closed once this PR is merged?

Reply:
I got another commit almost done that ups the max size too except for when using the pdf export. So, yes I'll make sure to tag that issue in the final commit so it gets marked done. Thanks!